cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/ApplyLogic.java

```diff
@@ -41,6 +41,11 @@
 
     private DiffLogic diffLogic;
 
+    private SpaceDevelopersOperations spaceDevelopersOperations;
+    private ServicesOperations servicesOperations;
+    private ApplicationsOperations applicationsOperations;
+
```

Comment:
Adding the SpaceOperations here would make the needed operations instances complete. Currently, they are just passed to the `applySpace(...)` method, but also having them as a field would be more consistent.

Reply:
Yeah definitely.